scss/components/_button.scss

```diff
@@ -310,3 +310,47 @@ table.button.warning:hover {
     }
   }
 }
+
+// Hollow Buttons
+@mixin hollowButton($color,$class:false) {
+    // If this mixin is called with a class, then a class will be added
+    // Then the mixin calls itself with just the color to obtain the color styles
+    @if $class {
+        &.#{$class} {
+            @include ghostButton($color);
```

Comment:
Damn! I renamed everything from Ghost to Hollow in order to follow the Sites framework. I will fix.